packages/protocol/contracts/rrp/authorizers/Api3TokenLock.sol

```diff
@@ -0,0 +1,419 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+import "./Api3RequesterRrpAuthorizer.sol";
+import "./interfaces/IApi3Token.sol";
+import "./interfaces/IApi3TokenLock.sol";
+import "../../admin/MetaAdminnable.sol";
+import "../../admin/RankedAdminnable.sol";
+
+/// @title The contract used to lock API3 Tokens in order to gain access to Airnodes
+contract Api3TokenLock is RankedAdminnable, MetaAdminnable, IApi3TokenLock {
+    enum AdminRank {
```

Comment:
I think the convention is to put enums in the interface (IApi3TokenLock in this case). Not a big deal to me personally tho.

Reply:
Its how it was done in the Whitelister.sol so I just replicated that